packages/cubejs-mssql-driver/driver/MSSqlDriver.js

```diff
@@ -1,6 +1,13 @@
 const sql = require('mssql');
 const { BaseDriver } = require('@cubejs-backend/query-orchestrator');
 
+const GenericTypeToMSSql = {
+  string: 'nvarchar(max)',
+  text: 'nvarchar(max)',
+  // should this be timezone aware? use datetimeoffset
+  timestamp: 'datetime',
```

Comment:
I am thinking about `datetime2`, but it's supported in MSSQL >= 2008

Reply:
@ovr I think you're right that `datetime2` would be a better option. It seems to be the suggested type from Microsoft going forward. This is from their documentation: 

> Use the time, date, datetime2 and datetimeoffset data types for new work. These types align with the SQL Standard. They are more portable. time, datetime2 and datetimeoffset provide more seconds precision. datetimeoffset provides time zone support for globally deployed applications.

I will update the source branch to use `datetime2`. Thanks for the review.